AutoscoperM/AutoscoperM.py

```diff
@@ -1194,3 +1281,62 @@ def convertNodeToData(self, volumeNode: slicer.vtkMRMLVolumeNode) -> vtk.vtkImag
             imageData = imageReslice.GetOutput()
 
         return imageData
+
+    def check4D(self, node: slicer.vtkMRMLNode) -> bool:
+        """
+        Checks if the volume is 4D
+        """
+        self.is_4d = type(node) == slicer.vtkMRMLSequenceNode
+
+    def getItemInSequence(self, sequenceNode: slicer.vtkMRMLSequenceNode, idx: int) -> slicer.vtkMRMLNode:
```

Comment:
Function that do not internally use `self` should be decorated with `@staticmethod`